BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -2380,6 +2384,8 @@ private void LaserBeam(string vesselname)
                                                     p.rb.AddForceAtPosition((tf.position - p.transform.position).normalized * (float)Impulse, p.transform.position, ForceMode.Acceleration);
                                                 }
                                                 if (BDArmorySettings.DEBUG_WEAPONS) Debug.Log($"[BDArmory.ModuleWeapon]: Impulse of {Impulse} Applied to {p.vessel.GetName()}");
+                                                //if (laserDamage == 0) 
+                                                    damage += Impulse / 100;
```

Comment:
For stock Gravgun, results in 50 'damage' per shot, or about on par with a Browning, which should at least let gravgun craft keep up (at least to an extent) with normal weapons. We can always adjust this later if needs be.

Reply:
Sounds good.